docs/reference/units/manifest-entry.rst

```diff
@@ -114,27 +116,64 @@ depends on data from the manifest can look like this::
     imports: from com.canonical.plainbox import manifest
 
 Note that the job uses the ``manifest`` job from the
-``com.canonical.plainbox`` name-space. It has to be imported using the
-``imports:`` field as it is in a different name-space than the one the example
-unit is defined in (which is arbitrary). Having that resource it can then check
+``com.canonical.plainbox`` namespace. It has to be imported using the
+``imports:`` field as it is in a different namespace than the one the example
+unit is defined in (which is arbitrary). Having that resource, it can then check
 for the ``has_thunderbolt`` field manifest entry in the
-``com.canonical.checkbox`` name-space. Note that the name-space of the
+``com.canonical.checkbox`` namespace. Note that the namespace of the
 ``manifest`` job is not related to the ``manifest.ns`` value. Since any
 provider can ship additional manifest entries and then all share the flat
-name-space of resource attributes looking at the ``.ns`` attribute is a way to
+namespace of resource attributes looking at the ``.ns`` attribute is a way to
 uniquely identify a given manifest entry.
 
 Collecting Manifest Data
 ------------------------
 
-To interactively collect manifest data from a user please include this job
-somewhere early in your test plan:
-``com.canonical.plainbox::collect-manifest``.
+When running Checkbox, if some jobs in the selected test plan depend on a
+manifest entry, a System Manifest screen will be presented so that the user
```

Comment:
Perhaps we should mention the "silent mode" (auto-manifest file) in this reference as well?

Reply:
FIxed in [0184d06](https://github.com/canonical/checkbox/pull/736/commits/0184d06489297f7ec4b2b659465b2fdd6a78de0d)